pom.xml

```
@@ -129,8 +129,8 @@
                 <version>2.3.1</version>
```

```suggestion
                <version>3.8.2</version>
```
Adding this , otherwise JDK 8 related codes end up showing as red in IDE.